### PR TITLE
PLF-7008 Stabilize Cluster

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: extension/config/src/main/resources/conf/platform/cache/infinispan/cluster/cache-config.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
         mode="replication">
       <stateTransfer
           timeout="${exo.cache.cluster.statetransfer.timeout:20000}"
-          fetchInMemoryState="${exo.cache.cluster.statetransfer.fetchinmemorystate:false}" />
+          fetchInMemoryState="${exo.cache.cluster.statetransfer.fetchinmemorystate:true}" />
```
Comment (fdrouet) [inline extension/config/src/main/resources/conf/platform/cache/infinispan/cluster/cache-config.xml]: For my knowledge, what is the purpose of this change ?

Comment (boubaker) [inline extension/config/src/main/resources/conf/platform/cache/infinispan/cluster/cache-config.xml]: To replicate the caches content from neighbors just after it's initialized.
This will evict to replicate the content value by value when it's requested via the API in the HTTP threads.

File: extension/config/src/main/resources/conf/platform/configuration.properties [hunk 2]
```diff
@@ -267,89 +294,6 @@ cache.exo.ecms.javascript.liveTime=${exo.cache.ecms.javascript.liveTime:3600}
 wcm.cache.clipboard.capacity=${exo.cache.ecms.clipboard.capacity:500}  
 wcm.cache.clipboard.timetolive=${exo.cache.ecms.clipboard.timetolive:-1}  
 
-
-# == SOCIAL Caches Configuration == #
-
-# Social Cache Configuration - Identity
-#  - Standalone (time to live in seconds)
-cache.exo.social.IdentityCache.Capacity=${exo.cache.social.IdentityCache.Capacity:500}
```
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: Gathering the properties in this file has been done to easily have all default values in one location. It is easier to check and to change them. Any reason why you want to remove them for social and calendar caches ?

Comment (boubaker) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: Yep, as discussed and explained in commit https://github.com/exodev/platform/pull/172/commits/23ad6d2b6630fd34d5770918a02e8fa73df3416b, we have to remove those properties in order to use the default value from XML that should be in milliseconds in cluster mode and in seconds in local cache mode.

Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: Right, I remember this discussion now, my bad :smile:

Comment (boubaker) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: no worry! thx @thomasdelhomenie ;-)
